.github/CODEOWNERS

```diff
@@ -170,7 +170,7 @@ packages/react-components/theme-designer @microsoft/cxe-coastal
 packages/react-components/global-context @microsoft/teams-prg
 packages/react-components/babel-preset-global-context @microsoft/teams-prg
 packages/react-components/react-table @microsoft/teams-prg
-## <%= NX-CODEOWNER-PLACEHOLDER %>
```

Comment:
this cannot be removed as it is used as a placeholder for automation. please add it back after the newly added codeowner line